### ISSUE TITLE
Does not work with jest 28

### ISSUE THREAD

**Issue by foray1010** [Connormiha/jest-css-modules-transform#51]
Received following error from jest 28.0.0 when importing css

```
Invalid return value:
      Code transformer's `process` method must return an object containing `code` key
      with processed string. If `processAsync` method is implemented it must return
      a Promise resolving to an object containing `code` key with processed string.
      Code Transformation Documentation:
      https://jestjs.io/docs/code-transformation
```

## Version

```
jest: 28.0.0
jest-css-modules-transform: 4.3.0
```

**Comment by joaopslins** [comment]
API for transformers has changed:

https://jestjs.io/docs/upgrading-to-jest28#transformer

**Comment by dackmin** [comment]
While waiting for a change you can get around this issue fairly easily:
```js
const transformer = require('jest-css-modules-transform');

const original = transformer.process;

transformer.process = (...args) => {
  const result = original(...args);

  return { code: result.code || result };
};

module.exports = transformer;
```

**Comment by foray1010** [comment]
@Connormiha Hi I saw that you supported jest 28 in v4.4.0, but it still does not work if injectIntoDOM flag is set
https://github.com/Connormiha/jest-css-modules-transform/blob/master/src/index.ts#L225

**Comment by Connormiha** [comment]
@foray1010 sorry for delay.
I fixed it in `4.4.1`

**Comment by foray1010** [comment]
It works perfectly! Thanks a lot!
